examples/jsm/renderers/webgpu/WebGPUBackend.js

```diff
@@ -114,6 +114,7 @@ class WebGPUBackend extends Backend {
 		this.adapter = adapter;
 		this.device = device;
 		this.context = context;
+		this.renderer._context = context;
```

Comment:
Instead of doing this, can't we just update `Renderer.getContext()` like so:
```js
getContext() {

	return this.backend.gl;

}

```

Reply:
Make sense! Updated and tested both Backend.

I'm using the second one:
```js
return this.backend.getContext();
```